x/staging/posts/types/keys.go

```diff
@@ -69,16 +61,21 @@ func PostStoreKey(id string) []byte {
 	return append(PostStorePrefix, []byte(id)...)
 }
 
-// PostIndexedIDStoreKey turns an id to a key used to store an incremental ID into the posts store
-func PostIndexedIDStoreKey(id string) []byte {
-	return append(PostIndexedIDStorePrefix, []byte(id)...)
-}
-
 // PostCommentsStoreKey turns an id to a key used to store a post's comments into the posts store
 func PostCommentsStoreKey(id string) []byte {
 	return append(PostCommentsStorePrefix, []byte(id)...)
 }
 
+// PostQuerySubspacePrefix returns the prefix used to store all the post id inside the subspace having the given id
+func PostQuerySubspacePrefix(subspace string) []byte {
+	return append(PostQueryStorePrefix, []byte(subspace)...)
+}
+
+// PostQueryStoreKey returns the store key to store the post id containing the given data
+func PostQueryStoreKey(subspace string, id string) []byte {
+	return append(PostQuerySubspacePrefix(subspace), []byte(id)...)
+}
```

Comment:
I think we can rename those to be more clear

```suggestion
// SubspacePostPrefix returns the prefix used to store all the posts present inside the subspace having the given id
func SubspacePostsPrefix(subspace string) []byte {
	return append(SubspacePostPrefix, []byte(subspace)...)
}

// SubspacePostKey returns the key used to associate the post with the given id to the subspace with the provided id
func SubspacePostKey(subspace string, id string) []byte {
	return append(PostQuerySubspacePrefix(subspace), []byte(id)...)
}
```